examples/jsm/controls/PointerLockControls.js

```diff
@@ -198,10 +198,14 @@ class PointerLockControls extends Controls {
 
 	/**
 	 * Activates the pointer lock.
+	 * 
+	 * @param {boolean} [unadjustedMovement=false] - Whether to disable OS-level adjustment for mouse acceleration and accesses raw mouse input instead or not.
```

Comment:
I am not a fan of double-negatives, but given that is the API, I'd try to be a clear as possible in the description here. Perhaps this:

>Disables OS-level adjustment for mouse acceleration, and accesses raw mouse input instead. The default value is false; setting it to true will disable mouse acceleration

Reply:
> The default value is false;

This term is not required in JSDoc anymore since the default value is automatically derived from `=false]`. The rest sounds good to me.